modules/tournament/src/main/TournamentMedleyUtil.scala

```diff
@@ -55,44 +55,46 @@ object TournamentMedleyUtil {
     val quick   = 0.9
     val fastest = 0.75
     Map(
-      "standard"      -> medium,
-      "chess960"      -> medium,
-      "kingOfTheHill" -> quick,
-      "threeCheck"    -> fastest,
-      "fiveCheck"     -> quick,
-      "antichess"     -> quick,
-      "atomic"        -> fastest,
-      "horde"         -> slow,
-      "racingKings"   -> fastest,
-      "crazyhouse"    -> quick,
-      "noCastling"    -> medium,
-      "monster"       -> quick,
-      "linesOfAction" -> fastest,
-      "scrambledEggs" -> fastest,
-      "frisian"       -> medium,
-      "frysk"         -> quick,
-      "international" -> slow,
-      "antidraughts"  -> slow,
-      "breakthrough"  -> slow,
-      "russian"       -> medium,
-      "brazilian"     -> medium,
-      "pool"          -> medium,
-      "portuguese"    -> medium,
-      "english"       -> medium,
-      "shogi"         -> slow,
-      "xiangqi"       -> medium,
-      "minishogi"     -> fastest,
-      "minixiangqi"   -> quick,
-      "flipello"      -> medium,
-      "flipello10"    -> slow,
-      "amazons"       -> medium,
-      "oware"         -> slow,
-      "togyzkumalak"  -> slow,
-      "go9x9"         -> medium,
-      "go13x13"       -> slow,
-      "go19x19"       -> slow,
-      "backgammon"    -> medium,
-      "nackgammon"    -> medium
+      "standard"               -> medium,
+      "chess960"               -> medium,
+      "kingOfTheHill"          -> quick,
+      "threeCheck"             -> fastest,
+      "fiveCheck"              -> quick,
+      "antichess"              -> quick,
+      "atomic"                 -> fastest,
+      "horde"                  -> slow,
+      "racingKings"            -> fastest,
+      "crazyhouse"             -> quick,
+      "noCastling"             -> medium,
+      "monster"                -> quick,
+      "linesOfAction"          -> fastest,
+      "scrambledEggs"          -> fastest,
+      "frisian"                -> medium,
+      "frysk"                  -> quick,
+      "international"          -> slow,
+      "antidraughts"           -> slow,
+      "breakthrough"           -> slow,
+      "russian"                -> medium,
+      "brazilian"              -> medium,
+      "pool"                   -> medium,
+      "portuguese"             -> medium,
+      "english"                -> medium,
+      "shogi"                  -> slow,
+      "xiangqi"                -> medium,
+      "minishogi"              -> fastest,
+      "minixiangqi"            -> quick,
+      "flipello"               -> medium,
+      "flipello10"             -> slow,
+      "amazons"                -> medium,
+      "oware"                  -> slow,
+      "togyzkumalak"           -> slow,
+      "go9x9"                  -> medium,
+      "go13x13"                -> slow,
+      "go19x19"                -> slow,
+      "backgammon"             -> medium,
+      "nackgammon"             -> medium,
+      "breakthroughtroyka"     -> fastest,
```

Comment:
I would make normal breakthrough quick, and leave mini to fastest. be good to differentiate them in case they end up in their own weighted medley.